float64 lati_value
float64 longi_value
float32 alti_value